### PR TITLE
Update SA1519 to examine using statements

### LINKED ISSUE #2180: SA1519 not detecting double using statement without braces
The following code does not produce ~~SA1503~~ SA1519 for the first `using` where it probably should:

``` C#
namespace Test
{
  using System.IO;

  public class TestClass
  {
    public void TestMethod()
    {
      using (var s = File.OpenRead("Test.data"))
      using (var b = new BinaryReader(s))
      {
        // ...
      }
    }
  }
}
```

### PR DESCRIPTION
**Pull request by sharwell**
Fixes #2180

### CODE REVIEW

File: StyleCop.Analyzers/StyleCop.Analyzers.Test/LayoutRules/SA1519UnitTests.cs [hunk 1]
```diff
@@ -120,6 +121,44 @@ public void Bar(int i)
...
+{
+    public void Bar(int i)
+    {
+        using (default(IDisposable))
+        using (default(IDisposable))
+        {
+        }
+    }
+}";
+            var fixedCode = @"using System.Diagnostics;
```
Comment (vweijsters) [inline StyleCop.Analyzers/StyleCop.Analyzers.Test/LayoutRules/SA1519UnitTests.cs]: This should be `using System;` (The AppVeyor build failed on this)

**Comment by vweijsters** [comment]
:+1: Looks good to me, except for the issue with the unit test

**Comment by sharwell** [comment]
@vweijsters There was also an indentation problem with the original code fix expectation. For some reason the braces for the second using statement do not get indented here.

**Comment by vweijsters** [comment]
:+1: What do you want to do with the wrong indentation? Make a new issue for it?

**Pull request merged**